fastify.d.ts

```diff
@@ -181,13 +182,7 @@ declare module '@fastify/error' {
   }
 }
 
-export interface ValidationResult {
-  keyword: string;
-  dataPath: string;
-  schemaPath: string;
-  params: Record<string, string | string[]>;
-  message: string;
-}
```

Comment:
Instead of using `ajv`, just amend the `ValidationResult` copying the different/new properties.